OpenSim/Simulation/Model/Appearance.h

```diff
@@ -0,0 +1,153 @@
+ * through the Warrior Web program.                                           *
+ *                                                                            *
+ * Copyright (c) 2005-2015 Stanford University and the Authors                *
+ * Author(s): Ayman Habib                                                     *
+ *                                                                            *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may    *
+ * not use this file except in compliance with the License. You may obtain a  *
+ * copy of the License at http://www.apache.org/licenses/LICENSE-2.0.         *
+ *                                                                            *
+ * Unless required by applicable law or agreed to in writing, software        *
+ * distributed under the License is distributed on an "AS IS" BASIS,          *
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.   *
+ * See the License for the specific language governing permissions and        *
+ * limitations under the License.                                             *
+ * -------------------------------------------------------------------------- */
+
+
+// INCLUDE
+#include <iostream>
+#include <OpenSim/Simulation/osimSimulationDLL.h>
+#include "SimTKcommon.h"
+#include <OpenSim/Common/Property.h>
+
+namespace OpenSim {
+
+class Body;
+class Model;
+
+
+//=============================================================================
+//=============================================================================
+/**
+ * A class that holds the Display Attributes (Appearance) of an object displayed 
+ * in the OpenSim Visualizer.
+ * 
+ * BaseAppearance contains Appearance properties that apply to all geometry.
+ * Geometry that have a surface so that it can be textured can use the subclass
+ * Appearance, while schematic line drawings (e.g. Arrows, Frames) can use 
+ * LineAppearance which offers thickness. 
+ *
+ * TODO: Add Resolution to Appearance from DecorativeGeometry and 
+ * utilize LineAppearance for Arrow, Line, and Frame (unused for now)
+ *
+ * @author Ayman Habib
+ * @version 1.0
+ */
+class OSIMSIMULATION_API BaseAppearance : public Object {
+    OpenSim_DECLARE_CONCRETE_OBJECT(BaseAppearance, Object);
+public:
+    //==========================================================================
+    // PROPERTIES
+    //==========================================================================
+    /** @name Property declarations
+    These are the serializable properties associated with Appearance. **/
+    /**@{**/
+    OpenSim_DECLARE_PROPERTY(color, SimTK::Vec3,
+        "The color (rgb) used to display the object. ");
+    OpenSim_DECLARE_PROPERTY(opacity, double,
+        "The opacity (0-1) used to display the object. ");
+    OpenSim_DECLARE_PROPERTY(representation, int,
+        "The representation (0:Hide, 1:Points, 2:Wire 2:Shaded) used to display the object. ");
```

Comment:
Would be more informative  to associate the names of the enum rather than integer value. "Hide" should probably be "Hidden"